modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java

```diff
@@ -154,6 +154,9 @@ public class RootProjectConfigurator implements Plugin<Project> {
 
 	public static final String DOWNLOAD_BUNDLE_TASK_NAME = "downloadBundle";
 
+	public static final String FORMAT_SOURCE_JAKARTA_TASK_NAME =
+		"formatSourceJakarta";
```

Comment:
Let's make this name `formatSourceJakartaTransform` everywhere to match the SF category name.